wbooks/src/redux/store.ts

```diff
@@ -1,18 +1,22 @@
-import { createStore, combineReducers, compose, applyMiddleware } from 'redux';
+import { createStore, combineReducers as CR, compose, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
+import { fetchMiddleware, wrapCombineReducers } from 'redux-recompose';
 
 import Reactotron from '../config/reactotronConfig';
 
 import book from './books/reducer';
 
-const reducers = combineReducers({
-  book
-});
+const middlewares = [thunk, fetchMiddleware];
+const enhancers = [applyMiddleware(...middlewares)];
 
-const enhancers = [applyMiddleware(thunk)];
+const combineReducers = wrapCombineReducers(CR);
```

Comment:
You don't need this. The invisible reducer should be used just for stuff like login, where you always want to do the same with that endpoint.
Actually, we never use invisible reducers in our projects.